### DISCUSSION TITLE
ability to "split down" entire secondary sidebar

### DISCUSSION THREAD

**Discussion by brndngrhm** [posit-dev/positron#3697] [Feedback]
really loving Positron so far!! I would love the ability to "split down" the entire secondary sidebar to be able to place the console underneath the variables, plots, session... tabs.

Currently i can drag the console underneath whichever secondary sidebar tab i have open (screenshot 1), but when i switch to another tab, the console remains on the previous tab (screenshot 2). I am able to achieve this setup in VS Code and would love to be able to do the same in Positron!

Screenshot 1: console underneath variables
<img width="926" alt="image" src="https://github.com/posit-dev/positron/assets/13240297/b672e6f2-da97-43d2-a1e9-43db25045919">

Screenshot 2: switch to Plots tab, console stays with the variables tab
![image](https://github.com/posit-dev/positron/assets/13240297/fd9e98a8-ca8d-4d59-8a95-4c1ee8433460)

**Comment by juliasilge** [comment]
Thanks for this feedback @brndngrhm! Do you mind sharing a screenshot of how you set up VS Code with an R interactive terminal?

**Reply by brndngrhm** [reply]
sure! see screenshots below for the set up I had in R studio and how i re-created it in VS Code. I also added a screen recording of the steps to get this set up.

Screenshot 1: R Studio pane set-up
<img width="2197" alt="Screenshot 2024-06-30 at 6 20 27 PM" src="https://github.com/posit-dev/positron/assets/13240297/5ceb8d52-315d-4416-b874-d1f891626c29">

Screenshot 2: re-created in VS Code
<img width="2111" alt="Screenshot 2024-06-30 at 6 07 49 PM" src="https://github.com/posit-dev/positron/assets/13240297/944da6a1-c032-4691-875d-637a013448f0">

VS Code steps:

https://github.com/posit-dev/positron/assets/13240297/568fbeb4-4703-4260-9fdd-f4e07a2137ac
